nltk/translate/bleu_score.py

```diff
@@ -49,14 +48,14 @@ def sentence_bleu(references, hypothesis, weights=[0.25, 0.25, 0.25, 0.25]):
     0.5045666840058485
 
     >>> sentence_bleu([reference1, reference2, reference3], hypothesis2)
-    0
+    0.39692877231857493
 
     The default BLEU calculates a score for up to 4grams using uniform
     weights. To evaluate your translations with higher/lower order ngrams, 
     use customized weights. E.g. when accounting for up to 6grams with uniform
     weights:
 
-    >>> weights = [0.1666, 0.1666, 0.1666, 0.1666, 0.1666]
+    >>> weights = (0.1666, 0.1666, 0.1666, 0.1666, 0.1666)
```

Comment:
It appears that [PEP-8](https://www.python.org/dev/peps/pep-0008/) is silent about this. My own preference is to retain the parens, but I don't feel strongly about it.